### ISSUE TITLE
Item set browse on site shows all results regardless of results per page setting

### ISSUE THREAD

**Issue by fitnycdigitalinitiatives** [omeka/omeka-s#2035]
Hi,

When you browse items sets on a site, it seems to show all items all at once regardless of what the results per page setting is (either for the site or globally). However, this is only the case when the page parameter is not set in the query. So if you go to `/item-set` it shows everything whereas `/item-set?page=1` respects the per page setting. I've confirmed this both on my own site and the omeka-s sandbox. But I can't seem to figure out where this bug occurs in the code. The site item-set controller seems identical to the item controller. This doesn't occur on the admin side, only for sites.

Without page parameter shows 8 sets even though limit is 5:
<img width="2559" alt="image" src="https://user-images.githubusercontent.com/12830594/225729528-033a9682-e00d-478f-b5a2-7dbb45859014.png">

Working correctly with page parameter:
<img width="2559" alt="image" src="https://user-images.githubusercontent.com/12830594/225729623-b9b94fbf-4925-4485-9149-3b90fa8fa32e.png">

Thanks,

Joseph Anderson

**Comment by zerocrates** [comment]
I'll have to go back and look at this...My recollection though is that this is intentional, since sets have been a "pick the specific ones you want" resource for sites for a longer time than items were, the logic being that a hand-selected group of sets should fit on a single page. Similarly, sets uniquely respect the specific order set for them in the Resources section of the site admin.

**Comment by fitnycdigitalinitiatives** [comment]
Ok, thanks, John. I guess though if that's the intention the presence of the [pagination](https://github.com/omeka/omeka-s/blob/8ba8610d3a997e5ce610335579ff37fc687ef0ce/application/view/omeka/site/item-set/browse.phtml#L18) on the item-set view does throw things off because it leads the user to think there are more pages, but it will just take you to another page duplicating the results.

Also, in our case (although not probably normal!) we have 150 item sets for one our sites.

**Comment by zerocrates** [comment]
Yeah there's definitely at least a mismatch with the pagination information that the pagination controls have vs. what's actually happening, so that's a bug regardless.

For simplicity and consistency we may just make this do pagination the same as other views... we'll see.

**Comment by allanaaa** [comment]
I am replicating this issue for every theme I've tested it on, @zerocrates @jimsafley. Since we are working on pagination issues. The text "1-5" is correctly showing the site setting, but the actual results table is ignoring that number. 

https://dev.omeka.org/amayer/amayer-s/omeka-s/s/my-omeka-s-site/item-set
https://dev.omeka.org/amayer/amayer-s/omeka-s/s/my-second-site/item-set
https://dev.omeka.org/amayer/amayer-s/omeka-s/s/my-third-site/item-set

**Comment by zerocrates** [comment]
Yeah so the situation is still as previously mentioned, the item set browse is basically not designed to paginate.

So we can either see the "bug" as being that the themes just shouldn't try to print pagination on the item set browse, or that the paginator should handle "actually this isn't paginated" better, or that the item set browse just _should_ paginate the same as everything else.

**Comment by alexdryden** [comment]
This has come up with our users as well. I submitted two possible solutions in a[ draft pull request](https://github.com/omeka/omeka-s/pull/2310) for this solution option: 

> or that the item set browse just should paginate the same as everything else.

One sets all of the defaults, same as admin side, the other just sets the page as 1 if it isn't set.
